app/kuma-dp/pkg/config/validate.go

```diff
@@ -27,7 +28,11 @@ func ValidateTokenPath(path string) error {
 		return errors.Wrapf(err, "could not read the token in the file %s", path)
 	}
 
-	token, parts, err := new(jwt.Parser).ParseUnverified(string(rawToken), &jwt.MapClaims{})
+	strToken := strings.TrimSpace(string(rawToken))
+	if !isASCII(strToken) {
+		return errors.New("The file cannot contain non ASCII character, avoid line breaks inside the token")
```

Comment:
I think I get what's meant but line breaks are ASCII. why is the `isASCII` check necessary, isn't a parse error enough?